Forwarded: part of Origin PR by Jojo-Schmitz
Origin: https://github.com/musescore/MuseScore/pull/9000
Author: Joachim Schmitz <jojo@schmitz-digital.de>
Description: Fix #333979: Crash when changing time signature on a 'hanging' slur

--- a/libmscore/range.cpp
+++ b/libmscore/range.cpp
@@ -696,7 +696,7 @@ bool ScoreRange::write(Score* score, con
                         }
                   else
                         s->setStartElement(0);
-                  if (slur->endCR()->isGrace()) {
+                  if (slur->endCR() && slur->endCR()->isGrace()) {
                         Chord* sc = slur->endChord();
                         int idx   = sc->graceIndex();
                         Chord* dc = toChord(score->findCR(s->tick2(), s->track2()));
